cubi_tk/irods/check.py

```diff
@@ -169,10 +188,10 @@ def execute(self):
             self.run_checks(data_objs)
             logger.info("All done")
 
-    def run_checks(self, data_objs: dict):
+    def run_checks(self, data_objs: typing.List[iRODSDataObject]):
         """Run checks on files, in parallel if enabled."""
-        num_files = len(data_objs["files"])
-        dsp_files = data_objs["files"]
+        num_files = len(data_objs)
+        dsp_files = data_objs
```

Comment:
Since we leave the output of `get_data_objs` as is, this should not need any chnages, right?